### PR TITLE
Fix inputs for grafana 8 0 3

### PR DESCRIPTION
**Pull request by th3M1ke**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -83,17 +83,18 @@ COPY --from=js-builder /usr/src/app/public ./public
 COPY --from=js-builder /usr/src/app/tools ./tools
 
 # LOGZ.IO GRAFANA CHANGE :: Copy custom.ini
+COPY custom.ini conf/custom.ini
 RUN cp "$GF_PATHS_HOME/conf/custom.ini" "$GF_PATHS_CONFIG"
 # LOGZ.IO GRAFANA CHANGE :: Preinstall plugins
-COPY ./data/plugins "$GF_PATHS_PLUGINS"
+COPY grafana/data/plugins "$GF_PATHS_PLUGINS"
 # LOGZ.IO GRAFANA CHANGE :: Remove news panel
 RUN rm -rf ./public/app/plugins/panel/news
```
Comment (avodotiiets) [inline Dockerfile:91]: Should be `grafana/public/app/plugins/panel/news`?

File: Dockerfile [hunk 2]
```diff
@@ -83,17 +83,18 @@ COPY --from=js-builder /usr/src/app/public ./public
...
 # LOGZ.IO GRAFANA CHANGE :: Remove pluginlist panel
 RUN rm -rf ./public/app/plugins/panel/pluginlist
```
Comment (avodotiiets) [inline Dockerfile:93]: Should be `grafana/public/app/plugins/panel/pluginlist`

File: Dockerfile [hunk 3]
```diff
@@ -2,18 +2,18 @@ FROM registry.internal.logz.io:5000/node:14.16.0-alpine3.13 as js-builder
 
 WORKDIR /usr/src/app/
 
-COPY package.json yarn.lock ./
-COPY packages packages
+COPY grafana/package.json grafana/yarn.lock ./
```
Comment (avodotiiets) [inline Dockerfile:5]: What is the reason of copying into `grafana` folder?

Comment (th3M1ke) [reply Dockerfile:5]: to be able to build like grafana 8.5.1

File: Dockerfile (hunk 2 above)
Comment (th3M1ke) [reply Dockerfile:93]: no, this is the correct path. we use these plugins https://github.com/logzio/data-viz/tree/logzio-v8.0.3/data/plugins

**Review by avodotiiets — APPROVED** [review]

**Pull request merged**